consensus/istanbul/backend/engine_test.go

```diff
@@ -1272,6 +1273,91 @@ func TestGovernance_Votes(t *testing.T) {
 				{vote{"governance.unitprice", uint64(2)}, 0}, // check on current
 			},
 		},
+		{
+			votes: []vote{
+				{"kip71.lowerboundbasefee", uint64(25000000000)},  // voted on block 1
```

Comment:
Should we vote to every block? it is not common practice in real use case.

Reply:
Yes, It is actually dummy data to create more blocks. It only checks data voted on block 2, 5.
votes on block 6~9 can be deleted.

Reply:
@kjhman21 Apparentely this test creates a block per each element in `votes`.
I have plan to simplify/refactor to [using map](https://github.com/klaytn/klaytn/blob/bd9db8914fdbf389f6fcc49fa79cc2173e3908f0/consensus/istanbul/backend/engine_test.go#L1009) some time later. But I'd say we keep it this way for now.

@kjeom not the best way, but you can use an empty `vote` to create a block without vote.
It works because return value of `engine.governance.AddVote(v.key, v.value)` is not checked.
It should make your intention more clear.

```
            votes: []vote{
                {"governance.governancemode", "none"},     // voted on block 1
                {"istanbul.committeesize", uint64(4)},     // voted on block 2
                {"governance.unitprice", uint64(2000000)}, // voted on block 3
                {"governance.governancemode", "single"},   // voted on block 4
                {},                                        // no votes on block 5
                {"governance.unitprice", uint64(2)},       // voted on block 6
                {"governance.governancemode", "none"},     // voted on block 7

```

Reply:
Empty vote entries looks better. thanks